packages/cli/src/deploy/resolveConfig.ts

```diff
@@ -22,18 +24,34 @@ export async function resolveConfig({
   readonly systems: readonly System[];
   readonly libraries: readonly Library[];
 }> {
-  const libraries = findLibraries(forgeOutDir).map((library): Library => {
-    // foundry/solc flattens artifacts, so we just use the path basename
-    const contractData = getContractData(path.basename(library.path), library.name, forgeOutDir);
-    return {
-      path: library.path,
-      name: library.name,
-      abi: contractData.abi,
-      prepareDeploy: createPrepareDeploy(contractData.bytecode, contractData.placeholders),
-      deployedBytecodeSize: contractData.deployedBytecodeSize,
-    };
+  const requirePath = await findUp("package.json");
+  if (!requirePath) throw new Error("Could not find package.json to import relative to.");
+  const require = createRequire(requirePath);
+
+  const moduleOutDirs = config.modules.flatMap((mod) => {
+    if (mod.artifactPath == undefined) {
+      return [];
+    }
+
+    // Navigate up two dirs to get the contract output directory
```

Comment:
I didn't mean to explain what `../..` does but _why_ we're specifically navigating up two dirs

```suggestion
    // forge artifacts are organized by `{outDir}/{sourceFilename}/{contractName}.json`, so we navigate up two directories to get to `outDir`
```